models/user.model.js

```diff
@@ -3,12 +3,21 @@ const { Schema } = mongoose;
 const uniqueValidator = require("mongoose-unique-validator");
 
 const userSchema = new Schema({
+  fullName: { type: String },
   username: { type: String, required: true },
-  email: { type: String, required: true, unique: true },
-  password: { type: String, required: true },
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+    match: [
+      /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/,
```

Comment:
please shift these validation String inside utils files.

Reply:
validation strings keeped in utils file.